.vscode/settings.json

```diff
@@ -3,16 +3,18 @@
     "appleboy",
     "browserstack",
     "callout",
-    "Downey",
     "dropdowns",
     "FDROID",
     "fontawesome",
     "fortawesome",
     "nightwatch",
     "perrywinkle",
     "PGPASSWORD",
-    "podverse",
     "scrollbars",
-    "stylelint"
+    "stylelint",
+    "Creonopoulos",
+    "Downey",
+    "FOSS",
+    "Podverse"
```

Comment:
There was some re-ordering here just because I had two branches in flight and when I rebased I picked the wrong order.